.gitignore

```diff
@@ -21,3 +21,8 @@ CMakeCache.txt
 
 *.ninja
 .ninja*
+install_manifest.txt
+googletest
+llvm-*/
+lib/symbioticpy/symbiotic/versions.py
```

Comment:
Could you split this change into a separate commit please? So that it's easier to revert this change if we would ever had a reason to do that.

Reply:
Since I'll drop changes in the README file, this change will be separated.